src/rgw/rgw_sal_motr.cc

```diff
@@ -1508,6 +1516,7 @@ int MotrObject::MotrDeleteOp::delete_obj(const DoutPrefixProvider* dpp, optional
     return rc;
   }
 
+
```

Comment:
Is this really needed? If not - please remove.

Reply:
Removed